From phil@ipom.com Tue Feb 19 02:16:25 2008
From: Konstantin Kletschke <konsti@ku-gbr.de>
Date: Tue, 19 Feb 2008 02:15:32 -0800
Subject: USB: storage: Nikon D80 new FW still needs Fixup
To: Greg KH <greg@kroah.com>
Cc: USB Dev <linux-usb@vger.kernel.org>,  USB Storage list <usb-storage@lists.one-eyed-alien.net>, Konstantin Kletschke <konsti@ku-gbr.de>
Message-ID: <47BAAC44.7010805@ipom.com>


From: Konstantin Kletschke <konsti@ku-gbr.de>

Add new BCD numbers for Nikon D80 Firmware revision v1.10 to the
unusual_devs.h file.

Signed-off-by: Konstantin Kletschke <konsti@ku-gbr.de>
Signed-off-by: Phil Dibowitz <phil@ipom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/unusual_devs.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/storage/unusual_devs.h
+++ b/drivers/usb/storage/unusual_devs.h
@@ -357,7 +357,7 @@ UNUSUAL_DEV(  0x04b0, 0x040f, 0x0100, 0x
 		US_FL_FIX_CAPACITY),
 
 /* Reported by Emil Larsson <emil@swip.net> */
-UNUSUAL_DEV(  0x04b0, 0x0411, 0x0100, 0x0101,
+UNUSUAL_DEV(  0x04b0, 0x0411, 0x0100, 0x0110,
 		"NIKON",
 		"NIKON DSC D80",
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
